numpy/random/_mt19937.pyx

```diff
@@ -226,6 +226,24 @@ cdef class MT19937(BitGenerator):
         -------
         bit_generator : MT19937
             New instance of generator jumped iter times
+
+        Notes
+        -----
+        The jump step is computed using a modified version of Matsumoto's
+        implementation of Horner's method. The step polynomial is precomputed
+        to perform 2**128 steps. The jumped state has been verified to match
+        the state produced using Matsumoto's original code.
+
+        References
+        ----------
+        .. [1] Matsumoto, M, Generating multiple disjoint streams of
```

Comment:
I think this reference might be the one causing a problem. Does the title need to be in quotes? Is the `[online]` bit standard?

Reply:
Also seems to be different indentation in the two references, but I don't think that should matter.

Reply:
Might have been spacing, or the `[Online]`?